src/target/arc.c

```diff
@@ -1633,9 +1630,9 @@ static int arc_unset_breakpoint(struct target *target,
 			LOG_ERROR("Invalid breakpoint length: target supports only 2 or 4");
 			return ERROR_COMMAND_ARGUMENT_INVALID;
 		}
-		breakpoint->is_set = false;
 
-	}	else if (breakpoint->type == BKPT_HARD) {
+		breakpoint->is_set = false;
```

Comment:
This looks like a different fix. Usually, u create one patch for each issue.

Reply:
Corrected, thank you.